package.json

```diff
@@ -58,13 +64,15 @@
   },
   "scripts": {
     "build:dev": "cross-env NODE_ENV=development webpack --config webpack.config",
-    "build:prod": "cross-env NODE_ENV=production webpack --config webpack.config",
+    "build:prod": "npm run test && cross-env NODE_ENV=production webpack --config webpack.config",
     "dev": "npm run build:dev && npm start",
-    "start": "node server"
+    "start": "node server",
+    "test": "mocha test/**/*.spec.js -r test/setup.js -r ignore-styles -R nyan"
   },
-  "license": "MIT",
   "dependencies": {
     "react-css-themr": "^1.6.0",
-    "tinycolor2": "^1.4.1"
+    "tinycolor2": "^1.4.1",
```

Comment:
I can see your point here, will pull all of these over to dev dependencies as once it's been bundled then you don't require any of these.